include/lsst/meas/extensions/psfex/PsfexPsf.h

```diff
@@ -97,6 +97,20 @@ class PsfexPsf : public lsst::afw::table::io::PersistableFacade<PsfexPsf>,
         lsst::afw::image::Color const& color       ///< colour of object
     ) const;
 
+    /// Compute the bbox of the kernel image at the specified position/color
+    virtual lsst::afw::geom::Box2I doComputeBBox(
```

Comment:
Yes, all the `doComputeBBox`s implemented on this ticket are independent of color but include it as a default parameter to mimic  `doComputeKernelImage`.  Correct me if I'm wrong but from what I can tell, none of the PSFs' `doComputeKernelImage` methods are color-dependent either, so including color in the Psf API is in expectation of future color dependent PSFs.